src/fee-accounts/containers/overview/account-overview.component.ts

```diff
@@ -14,7 +14,7 @@ import { FeeAccount } from '../../models/pba-accounts';
 @Component({
   selector: 'app-prd-fee-accounts-component',
   templateUrl: './account-overview.component.html',
-})
+});
```

Comment:
this ; needs to be removed, complication failed it breaks the start up of start:ng